MAUI/Linear-Gauge/getting-started.md

```diff
@@ -249,4 +249,4 @@ The following code example gives you the complete code of above configurations.
 
 ![.NET MAUI Linear Gauge](images/getting-started/maui-linear-gauge.PNG)
 
-N> View [`sample`](https://github.com/SyncfusionExamples/MAUI-Linear-Gauge-Getting-started-) in GitHub
\ No newline at end of file
+N> You can refer to our [.NET MAUI Linear Gauge](https://www.syncfusion.com/maui-controls/maui-linear-gauge) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Scheduler Example](https://github.com/syncfusion/maui-demos) that shows you how to render the Scheduler in .NET MAUI.
```

Comment:
Use the below demo link:
https://github.com/syncfusion/maui-demos/tree/master/MAUI/Scheduler

Reply:
Updated